packages/js/product-editor/src/components/editor/init-blocks.ts

```diff
@@ -1,12 +1,20 @@
+/**
+ * External dependencies
+ */
+import { registerCoreBlocks } from '@wordpress/block-library';
+
 /**
  * Internal dependencies
  */
 import { init as initName } from '../details-name-block';
 import { init as initSection } from '../section';
 import { init as initTab } from '../tab';
+import { init as initPricing } from '../pricing-block';
 
 export const initBlocks = () => {
+	registerCoreBlocks();
```

Comment:
Needed this to get the `column` blocks, although it does look like some of the blocks are already being registered somewhere, although I can't figure out where.
<img width="416" alt="Screenshot 2023-03-16 at 9 56 43 AM" src="https://user-images.githubusercontent.com/2240960/225565352-b2fad735-73d3-4066-bcad-c5a97676f957.png">